frontend/src/index.css

```diff
@@ -20,4 +20,38 @@ body {
 .container {
   margin: 0 auto;
   max-width: 1024px;
-}
\ No newline at end of file
+}
+
+/* CONTAINER TO CENTER A MESSAGE (almost) */
+.message-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vmin;
+}
+
+/* STICKY AND FIXED ELEMENTS */
+.sticky {
+  position: sticky;
+}
+
+.fixed {
+  position: fixed;
+}
+
+.sticky,
+.fixed {
+  width: 100%;
+  background-color: var(--bg-color-sticky, white);
+}
+
+.fixed-top,
+.sticky-top {
+  top: 0;
+}
+
+.fixed-bottom,
+.sticky-bottom {
+  bottom: 0;
+}
```

Comment:
Though I see what you did there, I'm not quite sure I'm a fan of it, since this feels like it encroaches inline-style territory. I'd appreciate if we could review this towards the end of the project

Reply:
Ich kann mit Beidem leben, solange wir nicht 10+ Klassen in den containern haben. Deswegen würde ich das vorerst so lassen und dann am Ende überlegen, ob das Sinn macht nochmal umzubauen, oder einfach so lassen